### PR TITLE
[CodeQuality]: Skip PHPDoc

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: rules-tests/CodeQuality/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/fqcn_in_namespace.php.inc [hunk 1]
```diff
@@ -8,14 +8,13 @@ class FqcnInNamespace
 {
     public function run()
     {
-        /** @var stdClass|null $stdClass */
```
Comment (staabm) [inline rules-tests/CodeQuality/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/fqcn_in_namespace.php.inc:11]: turned this positive test into a native return type... hopefully the test still does what it is expected todo

File: rules-tests/CodeQuality/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/skip_phpdoc.php.inc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace Rector\Tests\CodeQuality\Rector\Identical\FlipTypeControlToUseExclusiveTypeRector\Fixture;
+
+use Rector\Tests\CodeQuality\Rector\Identical\FlipTypeControlToUseExclusiveTypeRector\Source\stdClass;
+
+class SkipPhpDoc
+{
+    public function run()
+    {
+        /** @var stdClass|null $stdClass */
```
Comment (staabm) [inline rules-tests/CodeQuality/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector/Fixture/skip_phpdoc.php.inc:11]: the same test as above, but with phpdoc which should be ignored

**Comment by staabm** [comment]
I start working thru all `getType()` calls within the rules/ folder. I think in the end we should use `getNativeType()` in most places instead

**Comment by TomasVotruba** [comment]
Clear improvement. Thank you :clap:

**Comment by samsonasik** [comment]
@staabm for note: rector-downgrade-php works with `getType()` on purpose afaik, as the process can be multiple rules in the way:

- downgade parameter type widening to apply typed param to `@param` type, also, just because it use `@param`, it doesn't mean the type is not mixed in the vendor.
- downgrade the code itself that already use `@param`

see https://getrector.com/demo/6dcbb10c-e9fc-445c-9282-c5b4a596202c
tested at https://github.com/rectorphp/rector-downgrade-php/blob/main/tests/Issues/IssueDowngradeParamType/FixtureDownToPhp71/downgrade_param_reflection_get_attribute.php.inc#L7

**Pull request merged**